cli/lsp/language_server.rs

```diff
@@ -1576,7 +1576,6 @@ impl Inner {
         Ok(Some(text_edits))
       }
     } else {
-      self.client.show_message(MessageType::WARNING, format!("Unable to format \"{specifier}\". Likely due to unrecoverable syntax errors in the file."));
```

Comment:
We should replace it with `lsp_warn!()` so it shows up in the output panel

Reply:
It already logs a warning for the error on line 1562 (which includes the file path), so it seemed like another warning didn't really provide any value. For instance, after this PR you'll still get a warning like below in the lsp output.
![Screenshot 2024-04-02 at 6 04 56 PM](https://github.com/denoland/deno/assets/17734409/1958d388-2449-4433-a752-44c2d5cd9e8d)